src/ui-shell/header/header-navigation.component.ts

```diff
@@ -11,7 +11,7 @@ import { NavigationItem } from "./header-navigation-items.interface";
 	selector: "ibm-header-navigation",
 	template: `
 		<nav class="cds--header__nav" [attr.aria-label]="ariaLabel">
-			<ul class="cds--header__menu-bar" role="menubar">
+			<div class="cds--header__menu-bar" role="list">
```

Comment:
Switch to menubar

Reply:
Not switching to menubar as the pattern is not [necessary for most sites that are styled to look like a menubar](https://www.w3.org/WAI/ARIA/apg/example-index/menubar/menubar-navigation). We have expandable flyouts or sections but these [submenus are indicated visually by an icon & have the appropriate `aria-expanded`](https://www.w3.org/WAI/tutorials/menus/flyout/).